docs/examples/trees_example.py

```diff
@@ -0,0 +1,232 @@
+import trees
+from fury import window, actor, lib
+import numpy as np
+
+
+# TESTING A QUADTREE OF RANDOM 2D POINTS
+br = trees.Branch2d(4, 0.0, 1.0, 0.0, 1.0)
+
+npoints = 5
+
+points = np.random.rand(npoints, 2)
+
+tree = trees.Tree2d(br)
+
+points_dic = {}
+for i in range(npoints):
+    points_dic[tree.add_point(points[i])] = points[i]
+
+print("Points and its ids")
+print(points_dic)
+print()
+
+
+actorslist = trees.actor_from_branch_2d(tree.root(), (1.0, 1.0, 1.0), 1.0)
+zeros = np.zeros((npoints, 1))
+actorPoints = actor.dots(np.hstack((tree.root().all_points_list(), zeros)), (1.0, 0.5, 0.4), 1, 5)
+
+scene = window.Scene()
+scene.set_camera(position=(-6, 5, -10),
+                 focal_point=(tree.root().x_mid_point(),
+                              tree.root().y_mid_point(),
+                              0.0),
+                 view_up=(0.0, 0.0, 0.0))
+showmanager = window.ShowManager(
+    scene,
+    "trees demo",
+    (1920,
+     1080),
+    reset_camera=True,
+    order_transparent=True)
+
+scene.add(actorPoints)
```

Comment:
It would be great if all the tree structures would appear in the same screen instead of creating new scenes for each of them separately. If you use a grid of plots, with them side by side, maybe rotating along their axes.